orchestrator/api/activity.py

```
@@ -62,6 +66,59 @@ async def get_activity_feed(
+class DigestFeedbackRequest(BaseModel):
+    state_hash: str
+    rating: int  # -1 (down) or 1 (up)
+
+
+@router.post(
+    "/digest/feedback",
+    dependencies=[Depends(require_workspace_permission("agents:execute"))],
+)
+async def submit_digest_feedback(
+    body: DigestFeedbackRequest,
+    db: Session = Depends(get_db),
```

Ruff (0.15.21)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)